apps_product_creator/data/product_attribute.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--Copyright (C) 2017-Today: Odoo Community Association (OCA)-->
```

Comment:
I think this data should be demo and not data. This is custom implementation. 

like : https://github.com/OCA/interface-github/blob/10.0/github_connector/demo/github_organization_serie.xml